sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -351,6 +353,21 @@ def _make_metric_list_to_collect(self, custom_metrics):
                 cfg = {'name': name, 'table': table, 'column': column, 'instance_name': db_name, 'tags': tags}
                 metrics_to_collect.append(self.typed_metric(cfg_inst=cfg, table=table, column=column))
 
+        # Load database files
+        # TODO: if self.dbm_enabled
+        for name, column, metric_fn in DATABASE_FILES_IO:
+            cfg = {'name': name, 'column': column, 'tags': tags}
+
+            # A wrapper around the metric function is required because `metric_fn` is mutable on each loop
```

Comment:
If anyone can suggest a prettier way to do this, I'm all ears!

Reply:
Rather than caching the unbound method itself, save the name then `getattr(self, name)`

Reply:
Ah good suggestion!